src/main/java/javax/jmdns/impl/DNSRecord.java

```diff
@@ -567,31 +555,29 @@ public ServiceEvent getServiceEvent(JmDNSImpl dns) {
         protected void toString(final StringBuilder sb) {
             super.toString(sb);
             sb.append(" alias: '")
-                .append(_alias != null ? _alias.toString() : "null")
-                .append('\'');
+                    .append(alias)
+                    .append('\'');
         }
-
     }
 
     public static class Text extends DNSRecord {
-        // private static Logger logger = LoggerFactory.getLogger(Text.class);
-        private final byte[] _text;
+        private final byte[] bytes;
```

Comment:
i am not a fan of renaming the byte array to bytes instead of text, semantically speaking.
in my view more information was carried when it was called after what it contains, not what it is.

Reply:
I've also struggled how to name it, but I'm open for better suggestions. However generally speaking I share the opinion that the field name should not duplicate the name of its containing class.
https://rules.sonarsource.com/java/type/Code%20Smell/RSPEC-1700/

Of course if this is a blocker of this PR I happy to revert it back to `text`.